sdk/src/events/fetchLogs.ts

```diff
@@ -78,3 +83,23 @@ function chunk<T>(array: readonly T[], size: number): T[][] {
 		.map((_, index) => index * size)
 		.map((begin) => array.slice(begin, begin + size));
 }
+
+export class LogParser {
```

Comment:
is this duplicate code? why'd ya need it?

Reply:
I had to change something about it so not the exact same but also this is for code that wants to parse events without going through the EventSubscriber class ... my missing event listener is using the fetchLogs method directly